pyzscaler/zia/locations.py

```diff
@@ -1,21 +1,29 @@
-from restfly.endpoint import APIEndpoint
 from box import BoxList
-from pyzscaler.utils import snake_to_camel
+from restfly.endpoint import APIEndpoint
+
+from pyzscaler.utils import Iterator, snake_to_camel
 
 
 class LocationsAPI(APIEndpoint):
-    def list_locations(self):
+    def list_locations(self, **kwargs):
         """
-        Returns a list of configured locations.
+        Return list of locations.
```

Comment:
Quality/Grammar: Returns a list of locations.

Reply:
I agree that we should use the more "imperative-style" of docstrings. But that did not seem to be the case in the rest of the code..